core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/machineLearning/sklearnAdvanced/KNNTrainer/SklearnAdvancedKNNParameters.java

```diff
@@ -0,0 +1,31 @@
+
+package edu.uci.ics.texera.workflow.operators.machineLearning.sklearnAdvanced.KNNTrainer;
+
+import edu.uci.ics.texera.workflow.operators.machineLearning.sklearnAdvanced.base.ParamClass;
+
+public enum SklearnAdvancedKNNParameters implements ParamClass {
+    n_neighbors("n_neighbors", "int"),
+    p("p", "int"),
+    weights("weights", "str"),
+    algorithm("algorithm", "str"),
+    leaf_size("leaf_size", "int"),
+    metric("metric", "int"),
+    metric_params("metric_params", "str"),
+    n_jobs("n_jobs", "int"),;
```

Comment:
I suggest remove `n_jobs` as it may use joblib internally (please verify). we want to avoid usage of `joblibs` as it can cause zombie processes. It is fine for a train operator to be slow.

Reply:
Removed.